code/game/objects/items/stacks/cable_coil.dm

```diff
@@ -118,23 +118,24 @@
     
 /obj/item/stack/cable_coil/attack(mob/M as mob, mob/user as mob)
 	if(hasorgans(M))
```

Comment:
this is conceptually haslimbs(), but for now it should stay like this given theres plans to give non-humans limbs at some point.